datamade/la-metro-councilmatic
# Update Robert Garcia's headshot

## Overview

See title.

### Checklist


## Testing Instructions

 * Deploy to production and confirm Robert Garcia's headshot has updated.
 
Handles #681
